plugins/auth-backend-module-okta-provider/src/resolvers.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  createSignInResolverFactory,
+  OAuthAuthenticatorResult,
+  PassportProfile,
+  SignInInfo,
+} from '@backstage/plugin-auth-node';
+
+/**
+ * Available sign-in resolvers for the Okta auth provider.
+ *
+ * @public
+ */
+export namespace oktaSignInResolvers {
+  /**
+   * Looks up the user by matching their Okta username to the entity name.
+   */
+  export const usernameMatchingUserEntityName = createSignInResolverFactory({
```

Comment:
Question: There was a `emailMatchingUserEntityAnnotation` resolver on the old provider file... Just confirming, does that no longer make sense 🙂?

Reply:
Is it covered by the common email based one perhaps? https://github.com/backstage/backstage/blob/1b3fa23361f1af7c6054f15f1e5e5ebd1710ea9f/plugins/auth-node/src/sign-in/commonSignInResolvers.ts#L29

Reply:
@camilaibs good shout! I used the `auth-backend-module-gitlab-provider` as a guide, so not sure it still makes sense to keep it. @freben I think it's covered by the `commonSignInResolvers`. 
I will revert to using emailMatchingUserEntityAnnotation to keep it congruous with the old provider